pom.xml

```diff
@@ -35,6 +35,14 @@
   <description>Kafka integrations component</description>
 
   <dependencies>
+    <dependency>
+      <groupId>javax.inject</groupId>
+      <artifactId>javax.inject</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.opentable.components</groupId>
+      <artifactId>otj-spring</artifactId>
```

Comment:
This is kind of unfortunate.

Reply:
Yes, but that's where AppInfo is. Unless you want me to fall back on ENV variables, which seems to defeat the purpose?

Reply:
Nah.